Error message for a <forward> without "name" attribute should be positioned better

**Describe the issue**
I made an error in my Frank configuration. I had a `<forward>` without a `name`, only a `path` attribute. When I started this configuration with the Frank!Runner, the adapter in the configuration was not shown. There was an error message in the Frank!Console: `Configuration [default] an unhandled exception occurred while loading configuration [default]: nl.nn.adapterframework.configuration.ConfigurationException: error during unmarshalling configuration from file [Configuration.xml] with digester-rules-file [digester-rules.xml] in element [forward]: (SaxException) line [14] column [39] : (InvocationTargetException): (NullPointerException)`, but that message was not attributed to the adapter. A Java stack trace with a NullPointerException appeared in the boot window of the Frank!Runner.

**Reporter**
Martijn Dirkse

<!-- If your issue is a bug or a problem an existing instance, please provide us the context below this line  -->
**To Reproduce**
Steps to reproduce the behavior:
1. Set up the Frank!Runner with the configuration shown below.
2. Start the Frank!Runner.
3. See the mentioned Java stack trace in the window that pops up.
4. Browse to localhost.
5. See that adapter `default` is not present.
6. Check that the shown error message appears, but not under the adapter `default`.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Configuration**
```
<Configuration
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:noNamespaceSchemaLocation="./frankdoc.xsd"
  name="default"
>
    <Adapter name="default">
        <Receiver>
            <JavaListener name="dummy"></JavaListener>
        </Receiver>
        <Pipeline firstPipe="parse">
            <Exit path="EXIT" state="success" />
            <pipe className="nl.nn.adapterframework.pipes.CsvParserPipe"
                name="parse" fileContainsHeader="xxx" fieldNames="ignoreHeader1,ignoreHeader2">
                <!-- <Forward name="success" path="EXIT" /> -->
                <Forward path="EXIT" />
            </pipe>
        </Pipeline>
    </Adapter>
</Configuration>```
```

Here is the Java stack trace:
```
DispatcherManagerImpl DEBUG java.lang.ClassNotFoundException when trying to load DispatcherManager using ClassLoader [sun.misc.Launcher$ExtClassLoader] that is parent of [sun.misc.Launcher$AppClassLoader]: nl.nn.adapterframework.dispatcher.DispatcherManagerImpl
DispatcherManagerImpl DEBUG java.lang.ClassNotFoundException when trying to load DispatcherManager using ClassLoader [sun.misc.Launcher$AppClassLoader] that is parent of [java.net.URLClassLoader]: nl.nn.adapterframework.dispatcher.DispatcherManagerImpl
DispatcherManagerImpl DEBUG java.lang.ClassNotFoundException when trying to load DispatcherManager using ClassLoader [java.net.URLClassLoader] that is parent of [org.apache.catalina.loader.ParallelWebappClassLoader]: nl.nn.adapterframework.dispatcher.DispatcherManagerImpl
2021-06-18 09:59:54,931 ERROR [main][] org.apache.commons.digester3.Digester End event threw exception
java.lang.reflect.InvocationTargetException: null
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.commons.beanutils.MethodUtils.invokeMethod(MethodUtils.java:282) ~[commons-beanutils-1.8.3.jar:1.8.3]
        at org.apache.commons.digester3.AbstractMethodRule.invoke(AbstractMethodRule.java:245) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.AbstractMethodRule.end(AbstractMethodRule.java:192) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.Digester.endElement(Digester.java:1128) [commons-digester3-3.2.jar:3.2]
        at org.apache.xerces.parsers.AbstractSAXParser.endElement(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.AbstractXMLDocumentParser.emptyElement(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:2.12.1-xml-schema-1.1]
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:2.12.1-xml-schema-1.1]
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:2.12.1-xml-schema-1.1]
        at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) [ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.commons.digester3.Digester.parse(Digester.java:1642) [commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.Digester.parse(Digester.java:1745) [commons-digester3-3.2.jar:3.2]
        at nl.nn.adapterframework.configuration.ConfigurationDigester.digestConfiguration(ConfigurationDigester.java:203) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.ConfigurationDigester.digest(ConfigurationDigester.java:168) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.Configuration.configure(Configuration.java:237) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.createAndConfigureConfigurationWithClassLoader(IbisContext.java:388) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.load(IbisContext.java:310) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.load(IbisContext.java:263) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:139) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:101) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.lifecycle.IbisApplicationServlet.init(IbisApplicationServlet.java:79) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at javax.servlet.GenericServlet.init(GenericServlet.java:158) [servlet-api.jar:4.0.FR]
        at org.apache.catalina.core.StandardWrapper.initServlet(StandardWrapper.java:1134) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:1089) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:983) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4902) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5211) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:706) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1133) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1866) [catalina.jar:9.0.39]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_252]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_252]
        at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.39]
        at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:112) [?:1.8.0_252]
        at org.apache.catalina.startup.HostConfig.deployDirectories(HostConfig.java:1045) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:429) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1576) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:309) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:936) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:843) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374) [catalina.jar:9.0.39]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_252]
        at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.39]
        at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:134) [?:1.8.0_252]
        at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.Catalina.start(Catalina.java:772) [catalina.jar:9.0.39]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342) [bootstrap.jar:9.0.39]
        at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473) [bootstrap.jar:9.0.39]
Caused by: java.lang.NullPointerException
        at java.util.Hashtable.get(Hashtable.java:364) ~[?:1.8.0_252]
        at nl.nn.adapterframework.pipes.AbstractPipe.registerForward(AbstractPipe.java:292) ~[ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.pipes.AbstractPipe$$FastClassBySpringCGLIB$$2e2d3d9a.invoke(<generated>) ~[?:?]
        at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at nl.nn.adapterframework.pipes.CsvParserPipe$$EnhancerBySpringCGLIB$$27e382ec.registerForward(<generated>) ~[?:?]
        ... 74 more
2021-06-18 09:59:54,948 ERROR [main][] org.apache.commons.digester3.Digester An error occurred while parsing XML from '(already loaded from stream)', see nested exceptions
nl.nn.adapterframework.xml.SaxException: line [14] column [39]
        at nl.nn.adapterframework.xml.SaxException.createSaxException(SaxException.java:73) ~[ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.ConfigurationDigester$1.createSAXException(ConfigurationDigester.java:131) ~[ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at org.apache.commons.digester3.Digester.endElement(Digester.java:1133) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.xerces.parsers.AbstractSAXParser.endElement(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.AbstractXMLDocumentParser.emptyElement(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.impl.XMLNSDocumentScannerImpl.scanStartElement(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:2.12.1-xml-schema-1.1]
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:2.12.1-xml-schema-1.1]
        at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:2.12.1-xml-schema-1.1]
        at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.XMLParser.parse(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source) ~[ibis-xerces-2.12.1-xml-schema-1.1.jar:?]
        at org.apache.commons.digester3.Digester.parse(Digester.java:1642) [commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.Digester.parse(Digester.java:1745) [commons-digester3-3.2.jar:3.2]
        at nl.nn.adapterframework.configuration.ConfigurationDigester.digestConfiguration(ConfigurationDigester.java:203) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.ConfigurationDigester.digest(ConfigurationDigester.java:168) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.Configuration.configure(Configuration.java:237) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.createAndConfigureConfigurationWithClassLoader(IbisContext.java:388) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.load(IbisContext.java:310) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.load(IbisContext.java:263) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:139) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:101) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.lifecycle.IbisApplicationServlet.init(IbisApplicationServlet.java:79) [ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at javax.servlet.GenericServlet.init(GenericServlet.java:158) [servlet-api.jar:4.0.FR]
        at org.apache.catalina.core.StandardWrapper.initServlet(StandardWrapper.java:1134) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:1089) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:983) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4902) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5211) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:706) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1133) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1866) [catalina.jar:9.0.39]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_252]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_252]
        at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.39]
        at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:112) [?:1.8.0_252]
        at org.apache.catalina.startup.HostConfig.deployDirectories(HostConfig.java:1045) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:429) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1576) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:309) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:936) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:843) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384) [catalina.jar:9.0.39]
        at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374) [catalina.jar:9.0.39]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_252]
        at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.39]
        at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:134) [?:1.8.0_252]
        at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930) [catalina.jar:9.0.39]
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.39]
        at org.apache.catalina.startup.Catalina.start(Catalina.java:772) [catalina.jar:9.0.39]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342) [bootstrap.jar:9.0.39]
        at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473) [bootstrap.jar:9.0.39]
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.commons.beanutils.MethodUtils.invokeMethod(MethodUtils.java:282) ~[commons-beanutils-1.8.3.jar:1.8.3]
        at org.apache.commons.digester3.AbstractMethodRule.invoke(AbstractMethodRule.java:245) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.AbstractMethodRule.end(AbstractMethodRule.java:192) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.Digester.endElement(Digester.java:1128) ~[commons-digester3-3.2.jar:3.2]
        ... 66 more
Caused by: java.lang.NullPointerException
        at java.util.Hashtable.get(Hashtable.java:364) ~[?:1.8.0_252]
        at nl.nn.adapterframework.pipes.AbstractPipe.registerForward(AbstractPipe.java:292) ~[ibis-adapterframework-core-7.7-20210617.094217.jar:7.7-20210617.094217]
        at nl.nn.adapterframework.pipes.AbstractPipe$$FastClassBySpringCGLIB$$2e2d3d9a.invoke(<generated>) ~[?:?]
        at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:92) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671) ~[spring-aop-4.3.25.RELEASE.jar:4.3.25.RELEASE]
        at nl.nn.adapterframework.pipes.CsvParserPipe$$EnhancerBySpringCGLIB$$27e382ec.registerForward(<generated>) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_252]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_252]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_252]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_252]
        at org.apache.commons.beanutils.MethodUtils.invokeMethod(MethodUtils.java:282) ~[commons-beanutils-1.8.3.jar:1.8.3]
        at org.apache.commons.digester3.AbstractMethodRule.invoke(AbstractMethodRule.java:245) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.AbstractMethodRule.end(AbstractMethodRule.java:192) ~[commons-digester3-3.2.jar:3.2]
        at org.apache.commons.digester3.Digester.endElement(Digester.java:1128) ~[commons-digester3-3.2.jar:3.2]
        ... 66 more
18-Jun-2021 09:59:55.352 INFO [main] org.apache.catalina.core.ApplicationContext.log Successfully created IAF-API with SpringBus [ff-api-bus]
18-Jun-2021 09:59:57.851 INFO [main] org.apache.catalina.startup.HostConfig.deployDirectory Deployment of web application directory [C:\Users\martijn\frank-runner\build\apache-tomcat-9.0.39\webapps\ROOT] has finished in [14,183] ms
18-Jun-2021 09:59:57.855 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-80"]
18-Jun-2021 09:59:57.864 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in [14233] milliseconds
```

**Additional Environment**
 - Frank!Runner.
 - Chrome.

**Additional Context**
Of course the shown Frank config is wrong. There is also a clear error message, but it appears in an unexpected spot. I would expect the error to be visible under the heading of the `default` adapter. If possible, no Java stack trace should appear in the Frank!Runner pop-up window. The adapter `default` of the example should be visible. It should be red and have the error message.

--- a/core/src/main/java/nl/nn/adapterframework/core/IPipe.java
+++ b/core/src/main/java/nl/nn/adapterframework/core/IPipe.java
@@ -68,21 +68,22 @@ public interface IPipe extends IConfigurable, IForwardTarget {
 	/**
 	 * Register a PipeForward object to this Pipe. Global Forwards are added
 	 * by the PipeLine. If a forward is already registered, it logs a warning.
+	 * @throws ConfigurationException 
 	 * @see PipeLine
 	 * @see PipeForward
 	 */
-	void registerForward(PipeForward forward);
+	void registerForward(PipeForward forward) throws ConfigurationException;
 
 	/**
 	 * Perform necessary action to start the pipe. This method is executed
-	 * after the {@link #configure()} method, for eacht start and stop command of the
+	 * after the {@link #configure()} method, for each start and stop command of the
 	 * adapter.
 	 */
 	void start() throws PipeStartException;
 
 	/**
 	 * Perform necessary actions to stop the <code>Pipe</code>.<br/>
-	 * For instance, closing JMS connections, dbms connections etc.
+	 * For instance, closing JMS connections, DBMS connections etc.
 	 */
 	void stop();
 }

--- a/core/src/main/java/nl/nn/adapterframework/extensions/esb/DirectWrapperPipe.java
+++ b/core/src/main/java/nl/nn/adapterframework/extensions/esb/DirectWrapperPipe.java
@@ -74,8 +74,8 @@ public class DirectWrapperPipe extends TimeoutGuardPipe {
 		}
 		PipeForward pf = new PipeForward();
 		pf.setName(PipeForward.SUCCESS_FORWARD_NAME);
-		eswPipe.registerForward(pf);
 		try {
+			eswPipe.registerForward(pf);
 			eswPipe.configure();
 			return eswPipe.doPipe(message, session);
 		} catch (Exception e) {

--- a/core/src/main/java/nl/nn/adapterframework/pipes/AbstractPipe.java
+++ b/core/src/main/java/nl/nn/adapterframework/pipes/AbstractPipe.java
@@ -252,32 +252,26 @@ public abstract class AbstractPipe extends TransactionAttributes implements IExt
 		return sb.toString();
 	}
 
-	/**
-	 * Register a PipeForward object to this Pipe. Global Forwards are added
-	 * by the PipeLine. If a forward is already registered, it logs a warning.
-	 * @see PipeLine
-	 * @see PipeForward
-	 */
 	@Override
 	@IbisDoc({"30"})
-	public void registerForward(PipeForward forward) {
-		PipeForward current = pipeForwards.get(forward.getName());
-		if (current==null){
-			pipeForwards.put(forward.getName(), forward);
-		} else {
-			if (forward.getPath()!=null && forward.getPath().equals(current.getPath())) {
-				ConfigurationWarnings.add(this, log, "has forward ["+forward.getName()+"] which is already registered");
+	public void registerForward(PipeForward forward) throws ConfigurationException {
+		String forwardName = forward.getName();
+		if(forwardName != null) {
+			PipeForward current = pipeForwards.get(forwardName);
+			if (current==null){
+				pipeForwards.put(forwardName, forward);
 			} else {
-				log.info(getLogPrefix(null)+"PipeForward ["+forward.getName()+"] already registered, pointing to ["+current.getPath()+"]. Ignoring new one, that points to ["+forward.getPath()+"]");
+				if (forward.getPath()!=null && forward.getPath().equals(current.getPath())) {
+					ConfigurationWarnings.add(this, log, "has forward ["+forwardName+"] which is already registered");
+				} else {
+					log.info(getLogPrefix(null)+"PipeForward ["+forwardName+"] already registered, pointing to ["+current.getPath()+"]. Ignoring new one, that points to ["+forward.getPath()+"]");
+				}
 			}
+		} else {
+			throw new ConfigurationException(getLogPrefix(null)+"has a forward without a name");
 		}
 	}
 
-	/**
-	 * Perform necessary action to start the pipe. This method is executed
-	 * after the {@link #configure()} method, for each start and stop command of the
-	 * adapter.
-	 */
 	@Override
 	public void start() throws PipeStartException {
 //		if (getTransactionAttributeNum()>0 && getTransactionAttributeNum()!=JtaUtil.TRANSACTION_ATTRIBUTE_SUPPORTS) {
@@ -290,10 +284,6 @@ public abstract class AbstractPipe extends TransactionAttributes implements IExt
 //		}
 	}
 
-	/**
-	 * Perform necessary actions to stop the <code>Pipe</code>.<br/>
-	 * For instance, closing JMS connections, dbms connections etc.
-	 */
 	@Override
 	public void stop() {}
 

--- a/core/src/test/java/nl/nn/adapterframework/pipes/CounterSwitchPipeTest.java
+++ b/core/src/test/java/nl/nn/adapterframework/pipes/CounterSwitchPipeTest.java
@@ -11,7 +11,7 @@ import nl.nn.adapterframework.core.PipeRunResult;
 public class CounterSwitchPipeTest extends PipeTestBase<CounterSwitchPipe> {
 
 	@Override
-	public CounterSwitchPipe createPipe() {
+	public CounterSwitchPipe createPipe() throws ConfigurationException {
 		CounterSwitchPipe pipe = new CounterSwitchPipe();
 		pipe.registerForward(new PipeForward("1", null));
 		pipe.registerForward(new PipeForward("2", null));

--- a/core/src/test/java/nl/nn/adapterframework/pipes/IsXmlIfPipeTest.java
+++ b/core/src/test/java/nl/nn/adapterframework/pipes/IsXmlIfPipeTest.java
@@ -15,7 +15,7 @@ public class IsXmlIfPipeTest extends PipeTestBase<IsXmlIfPipe> {
 	String pipeForwardElse = "else";
 
 	@Override
-	public IsXmlIfPipe createPipe() {
+	public IsXmlIfPipe createPipe() throws ConfigurationException {
 		IsXmlIfPipe isXmlIfPipe = new IsXmlIfPipe();
 
 		//Add default pipes

--- a/core/src/test/java/nl/nn/adapterframework/pipes/PgpPipeTest.java
+++ b/core/src/test/java/nl/nn/adapterframework/pipes/PgpPipeTest.java
@@ -113,7 +113,7 @@ public class PgpPipeTest {
 	 * Creates pipes and pipeline session base for testing.
 	 */
 	@Before
-	public void setup() {
+	public void setup() throws ConfigurationException {
 		session = new PipeLineSession();
 
 		encryptPipe = new PGPPipe();

--- a/core/src/test/java/nl/nn/adapterframework/pipes/PipeTestBase.java
+++ b/core/src/test/java/nl/nn/adapterframework/pipes/PipeTestBase.java
@@ -41,7 +41,7 @@ public abstract class PipeTestBase<P extends IPipe> {
 	@Rule
 	public ExpectedException exception = ExpectedException.none();
 
-	public abstract P createPipe();
+	public abstract P createPipe() throws ConfigurationException;
 
 	@Before
 	public void setup() throws Exception {

--- a/core/src/test/java/nl/nn/adapterframework/pipes/XmlIfTest.java
+++ b/core/src/test/java/nl/nn/adapterframework/pipes/XmlIfTest.java
@@ -16,7 +16,7 @@ public class XmlIfTest extends PipeTestBase<XmlIf>{
 	private String pipeForwardElse = "else";
 
 	@Override
-	public XmlIf createPipe() {
+	public XmlIf createPipe() throws ConfigurationException {
 		XmlIf xmlIf = new XmlIf();
 
 		//Add default pipes

--- a/core/src/test/java/nl/nn/adapterframework/pipes/XmlValidatorTest.java
+++ b/core/src/test/java/nl/nn/adapterframework/pipes/XmlValidatorTest.java
@@ -81,7 +81,7 @@ public class XmlValidatorTest extends XmlValidatorTestBase {
         return getUnconfiguredValidator(schemaLocation, false, implementation);
     }
 
-   public static XmlValidator getUnconfiguredValidator(String schemaLocation, boolean addNamespaceToSchema, Class<AbstractXmlValidator> implementation) {
+   public static XmlValidator getUnconfiguredValidator(String schemaLocation, boolean addNamespaceToSchema, Class<AbstractXmlValidator> implementation) throws ConfigurationException {
         XmlValidator validator = new XmlValidator();
         try {
             validator.setImplementation(implementation);

--- a/test-integration/src/test/java/nl/nn/adapterframework/ldap/LdapFindMemberPipeTest.java
+++ b/test-integration/src/test/java/nl/nn/adapterframework/ldap/LdapFindMemberPipeTest.java
@@ -30,7 +30,7 @@ public class LdapFindMemberPipeTest {
 	private LdapFindMemberPipe pipe;
 	
 	@Before
-	public void setUp() {
+	public void setUp() throws ConfigurationException {
 		pipe = new LdapFindMemberPipe();
 		pipe.registerForward(new PipeForward(PipeForward.SUCCESS_FORWARD_NAME, null));
 		pipe.setHost(host);

--- a/test-integration/src/test/java/nl/nn/adapterframework/ldap/LdapFindMembershipsPipeTest.java
+++ b/test-integration/src/test/java/nl/nn/adapterframework/ldap/LdapFindMembershipsPipeTest.java
@@ -35,7 +35,7 @@ public class LdapFindMembershipsPipeTest {
 	private LdapFindGroupMembershipsPipe pipe;
 	
 	@Before
-	public void setUp() {
+	public void setUp() throws ConfigurationException {
 		pipe = new LdapFindGroupMembershipsPipe();
 		pipe.registerForward(new PipeForward(PipeForward.SUCCESS_FORWARD_NAME,null));
 		pipe.setLdapProviderURL(ldapProviderUrl);

